docs/de/config/multihost.md

```diff
@@ -6,8 +6,60 @@ editLink:    "https://github.com/ioBroker/ioBroker.docs/edit/master/docs/config/
 
 # Der Multihost-Betrieb
 
-?> ***Dies ist ein Platzhalter***.
-   <br><br>
-   Hilf mit bei ioBroker und erweitere diesen Artikel.  
-   Bitte beachte den [ioBroker Style Guide](community/styleguidedoc), 
-   damit die Änderungen einfacher übernommen werden können.
+## Master Konfiguration
+
+- login on master
+- run following command:
+
+> iobroker setup custom
```

Comment:
This step is only required if user has redis as State or Object database.
Please write some notice about it.